source/usage-examples/distinct.txt

```diff
@@ -0,0 +1,48 @@
+==========================================
+Distinct Values of a Field in a Collection
+==========================================
+
+.. default-domain:: mongodb
+
+You can retrieve a list of distinct values for a field across a collection
+by using the
+`distinct()
+<https://mongodb.github.io/node-mongodb-native/3.4/api/Collection.html#distinct>`_
+method. The ``distinct()`` method is called on a collection object and
+requires a document field name parameter. When called on a collection,
+``distinct()`` produces a list that contains one of each of the different
+values found in the specified document field.
+
+You can provide an optional query document to narrow the set of documents that
+are searched and an optional Object to specify additional query parameters.
+For details on the optional parameters, see the
+`distinct() method in the API documentation
+<https://mongodb.github.io/node-mongodb-native/3.4/api/Collection.html#distinct>`_.
+
+The ``distinct()`` method returns a
+`Promise <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise>`_
+that resolves to an
+`Array <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array>`_
+that contains each of the different values. If none of the documents in the
+collection contain the field specified in the method call, it returns an
+empty Array.
+
+If you specify a value for the document field name that is not a String such
+as a document, array, number, or null, the method does not execute and
```

Comment:
I have used monospaced font when I mention ``null`` in these examples, since it technically is a specific value of itself. You could do the same here for consistency.

Reply:
Good point. That made me think about how I phrased this sentence which could be more clear to identify that String, document, etc. are types instead of values. I can update null to be monospaced, though we should probably record that decision somewhere so that we can ensure consistency before release and beyond. There's also the adjective form of null when used as "null value" that we should decide on outside of this PR.